packages/entities/entities-shared/src/components/entity-base-config-card/ConfigCardDisplay.vue

```diff
@@ -120,10 +120,18 @@ const jsonContent = ref('')
 const yamlContent = ref('')
 
 watch(() => props.format, (format: string) => {
+  if (format === 'structured') {
```

Comment:
```suggestion
  if (format !== 'structured') {
```

and do not return anything

Reply:
Updated